builtin/k8s/releaser.go

```diff
@@ -358,7 +358,9 @@ func (r *Releaser) resourceServiceDestroy(
 
 	step = sg.Add("Deleting service...")
 	if err := serviceclient.Delete(ctx, state.Name, metav1.DeleteOptions{}); err != nil {
-		return err
+		if !errors.IsNotFound(err) {
+			return err
+		}
```

Comment:
Maybe we should log the error otherwise? Just for debug purposes.

Reply:
Good idea, added in de591fb38